apps/vr-tests-web-components/src/stories/checkbox/checkbox-indeterminate.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28871&commitId=3fd397a5ad4b32358d9bc512b3921d91218db769&env=prod&clientType=FLUENTUI)


  <details><summary><b>Checkbox</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checked - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Checked.normal.chromium.png|0| Added|
|Checkbox.Circular - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Circular - RTL.normal.chromium.png|0| Added|
|Checkbox.Circular Checked - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Circular Checked.normal.chromium.png|0| Added|
|Checkbox.Circular Disabled - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Circular Disabled Checked - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Circular Disabled Checked.normal.chromium.png|0| Added|
|Checkbox.Circular Disabled.normal.chromium.png|0| Added|
|Checkbox.Circular Indeterminate - Dark Mode.indeterminate.chromium.png|0| Added|
|Checkbox.Circular Indeterminate - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Circular Indeterminate.indeterminate.chromium.png|0| Added|
|Checkbox.Circular Indeterminate.normal.chromium.png|0| Added|
|Checkbox.Circular.normal.chromium.png|0| Added|
|Checkbox.Default - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Default - RTL.normal.chromium.png|0| Added|
|Checkbox.Default.normal.chromium.png|0| Added|
|Checkbox.Disabled - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Disabled Checked - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Disabled Checked.normal.chromium.png|0| Added|
|Checkbox.Disabled.normal.chromium.png|0| Added|
|Checkbox.Indeterminate - Dark Mode.indeterminate.chromium.png|0| Added|
|Checkbox.Indeterminate - Dark Mode.normal.chromium.png|0| Added|
|Checkbox.Indeterminate.indeterminate.chromium.png|0| Added|
|Checkbox.Indeterminate.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>